Source/Applications/SystemCenter/Controllers/OpenXDA/OpenXDAControllers.cs

```diff
@@ -48,6 +48,48 @@
 
 namespace SystemCenter.Controllers.OpenXDA
 {
+    public class OpenXDAApiController
```

Comment:
I don't think this is necessary. It looks like this Class is only used in Line 515 for a single Get Call. See comments there, but I would remove this class.

Reply:
It is also used in line 558 in a separate controller

Reply:
I'd still move this locally into the controller. (a) it's easier to track what settings are used if they are directly in the controller using them and (b) it's really the `API` that should contain these methods and not a separate class only existing in SystemCenter if we were to move that way.